### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#178] [Esercizi - Bertini]
![Screenshot 2023-11-06 173915](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/617fa384-22d5-491e-ae2f-03b942ead18f)

**Comment by Elia-Belli** [comment]
**Definizione v.a indipendenti**: siano $X,Y$ due v.a, esse sono indipendenti se, presi comunque $x\in Im(X)$ e $y\in Im(Y)$ gli eventi $(X=x),(Y=y)$ sono indipendenti, ovvero $\mathbb{P}(X=x,Y=y)=\mathbb{P}(X=x)\cdot \mathbb{P}(Y=y)$.

- **(1)** Se $X$ è una v.a. certa ha $|Im(X)|=1$, in questo caso tutti gli elementi di $\Omega$ mappati da $X$ in $\mathbb{R}$ hanno stessa immagine $c\Rightarrow \mathbb{P}(X=c)=1$.
Adesso per mostrare che $Y$ è indipendente da $X$ ci basta mostrare che preso comunque $y \in Im(Y)$ gli eventi $(Y=y),(X=c)$ sono indipendenti, per farlo utilizzo la probabilità condizionata:
$$\mathbb{P}(Y=y|X=c)=\frac{\mathbb{P}(Y=y\cap X=c)}{\mathbb{P}(X=c)}=\frac{\mathbb{P}(Y=y)}{1}=\mathbb{P}(Y=y)$$
$$\mathbb{P}(X=c|Y=y)=\frac{\mathbb{P}(Y=y\cap X=c)}{\mathbb{P}(Y=y)}=\frac{\mathbb{P}(Y=y)}{\mathbb{P}(Y=y)}=1=\mathbb{P}(X=c)$$
Allora $X,Y$ sono indipendenti $\square$

> TODO: **(3)** Esempio delle v.a. $X=\cos x$ e $Y=\sin x$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/7dda79ab-be0f-4574-95c2-77a8df378ae5)

**Reply by UnoCasuale** [reply]
Per la seconda implicazione il testo diche che le variabili aleatorie sono binarie non 0,1 é quindi possibile che assumano valori generici "a" e "b" e che quindi il valore di attesa E(x) sia E(x) = a(1-p)+ bp? Perché in quel caso poi non potremmo basarci sul caso specifico di x e y uguali ad 1
